src/dr_log.cpp

```diff
@@ -67,6 +68,30 @@ namespace {
 		return stream;
 	}
 
+	/// Tag for giving syslog severity level in log output
+	struct SystemdSeverityTag;
+
+	/// Format a severity level into systemd severity level for log output.
+	boost::log::formatting_ostream & operator<< (boost::log::formatting_ostream & stream,  log::to_log_manip<LogLevel, SystemdSeverityTag> const & level) {
+		static char const * strings[] = {
+				SD_DEBUG,
+				SD_INFO,
+				SD_INFO,
+				SD_WARNING,
+				SD_ERR,
+				SD_CRIT,
+		};
```

Comment:
It's a bit of a shame to add a dependency just for these constants. Considering they're part of log output, I also don't believe systemd can change them in the future.

I'd prefer to just copy the constants and avoid the dependency on `libsystemd`.

Reply:
ack